### ISSUE TITLE
Reproduction of the results.

### ISSUE THREAD

**Issue by julyiii** [jkpark0825/EANet#2]
&ensp;First of all, thank you for your awesome work!
&ensp;I'm trying to reproduce your code and have several questions. Hope you  can clarify my confusion. 

1. &ensp;In your paper, you mention that "we trained our model for 30 epochs, with learning rate annealing at 10th and 15th epochs from the initial learning rate 1 × 10−4",  What is the value of the decay rate?
2. &ensp;Could you provide a more specific description of the loss, such as the weight?
3. &ensp;How many SJT  layer used in network?

**Comment by jkpark0825** [comment]
1. The decay rate is set at 0.1.
2. The loss weights for relative translation, mano pose, mano shape, joint
(camera coordinates), mano joint (camera coordinates), joint (pixel
coordination), mano joint (pixel coordination), mano joint projection, and
mesh are as follows: 1, 1, 1, 10, 10, 1, 1, 1, and 1, respectively.
3. We utilize 6 SJT layers for each hand (Total 12 SJT layers).

We appreciate your interest, and we plan to upload our code as soon as
possible.

Best regards,
Joonkyu Park

2023년 9월 11일 (월) 오후 9:41, julyiii ***@***.***>님이 작성:

**Comment by julyiii** [comment]
&ensp;I have encountered two other questions.

1. Will you use valid annotation to filter out invalid joints?
2. When the root joint of one hand is not unvalid, will it be filtered out (both joints and mano_vertices) like [here](https://github.com/facebookresearch/InterHand2.6M/blob/67ba1b8e2c8da0f79ba8a3de5bb401714b4ebea2/data/InterHand2.6M/dataset.py#L82C18-L82C19) in your setting during training or testing?

Looking forward your reply!

**Comment by jkpark0825** [comment]
1. We filter out invalid joints like InterHand2.6M repo.
2. Yes, we follow InterHand2.6M repo.

2023년 9월 13일 (수) 오전 4:04, julyiii ***@***.***>님이 작성:
